<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="bgwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#bgwidget{
background-color:qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0, stop:0 rgba(0, 60, 216, 255), stop:1 rgba(255, 255, 255, 255));}
</string>
   </property>
   <widget class="QLabel" name="text1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>-10</y>
      <width>571</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 87 38pt &quot;Arial Black&quot;;
 color:rgb(101, 116, 255);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;Crime Prediction&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="GraphtText">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>     Analysis Graph of kerala</string>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>721</width>
      <height>491</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>ppp.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="GraphBack">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>590</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;
background-color: rgb(255, 233, 166);
;
font: 75 16pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="pppp.qrc"/>
 </resources>
 <connections/>
</ui>
